DisplaySettings/DisplaySettings.json

```diff
@@ -1841,16 +2018,15 @@
                 },
                 "required": [
                     "audioPort",
-                    "soundMode",
-                    "persist"
+                    "soundMode"
                 ]
             },
             "result": {
                 "$ref": "#/definitions/result"
             }
         },
         "setSurroundVirtualizer":{
-            "summary": "Sets the Surround Virtualizer boost. The Speaker/Surround Virtualizer enables a surround sound signal (including one generated by the Surround Decoder) to be rendered over a device with built-in speakers or headphones.",
+            "summary": "(Version 1)Sets the Surround Virtualizer boost. The Speaker/Surround Virtualizer enables a surround sound signal (including one generated by the Surround Decoder) to be rendered over a device with built-in speakers or headphones.\n \n### Event \n\n No Events.",
```

Comment:
The use of `(Version 1)` is a new convention and a good idea. Consider updating all RDK services that have methods with multiple versions to use this convention.

Reply:
As discussed keeping both versions in JSON file and only version2 methods in md file.